src/icarus/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+Icarus
+"""
+
+from icarus.moosesetup import MooseSetup
+from icarus.datasetgenerator import DatasetGenerator
+from icarus.modelbuilder import ModelBuilder
+from icarus.userinterface import UserInterface
+
+__all__ = ["moosesetup",
+           "datasetgenerator",
+           "modelbuilder",
+           "userinterface"]
```

Comment:
Here you put the objects to export, not the module names, so it should be:
```suggestion
__all__ = ["MooseSetup",
           "DatasetGenerator",
           "ModelBuilder",
           "UserInterface"]
```